server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/CreateSystemIndicesIT.java

```diff
@@ -201,10 +225,44 @@ public void testCreateSystemAliasViaComposableTemplate() throws Exception {
             ).get()
         );
 
-        assertAcked(prepareCreate(PRIMARY_INDEX_NAME));
-        ensureGreen(PRIMARY_INDEX_NAME);
+        assertAcked(prepareCreate(primaryIndexName));
+        ensureGreen(primaryIndexName);
+    }
 
-        assertHasAliases(Set.of(".test-index", ".test-index-composable-alias"));
+    /**
+     * Check that a composable template does not create an alias for a system index
+     */
+    public void testCreateSystemAliasViaComposableTemplate() throws Exception {
+        createIndexWithComposableTemplates(INDEX_NAME, PRIMARY_INDEX_NAME);
+
+        assertHasAliases(Set.of(INDEX_NAME), INDEX_NAME, PRIMARY_INDEX_NAME, 1);
+
+        assertAcked(
```

Comment:
I haven't tried it locally but I bet we don't need this one either.

Reply:
I think just like the other one, it doesn't hurt.